Update better icon and fanart images

zag2me opened the pull request:
Improve the artwork

Commit:
Update better icon and fanart images

lwille commented:
Wow, this looks great! Sadly I can't just accept this PR without further licensing information.

I'm not even sure if it's legally ok at all to use the Udemy logo for this plugin, as it suggests that this is something "official". 

I'd be glad about some help/research/reaching out to Udemy on that matter.

zag2me committed:
[update] Use open pixio image for fanart

Image Source: https://pixnio.com/objects/computer/person-work-laptop-computer-hand-technology-pencil-man-internet

"Image is in public domain, not copyrighted, no rights reserved, free for any use. You can use picture for any personal and commercial use without the prior written permission and without fee or obligation."

zag2me commented:
No worries... updated with a public domain fanart image.

Image Source: https://pixnio.com/objects/computer/person-work-laptop-computer-hand-technology-pencil-man-internet

Icon should be fine, I redrew it in GIMP to get the high quality and added a nice gradient background. Should be no issue with using Udemy Logo as long as its labeled 3rd party Add-on in addon.xml. Same for every other Kodi Add-on really.

Thanks for the Add-on, just wanted to give something back ;)

lwille commented:
Thank you so much! The "old" icon was taken from Wikipedia (they have a traced SVG version).
I'm not worried about the original pixels created by Udemy, rather by using their trademark.
